scout/server/blueprints/variant/controllers.py

```diff
@@ -188,6 +188,11 @@ def variant(
 
     managed_variant = store.find_managed_variant_id(variant_obj["variant_id"])
 
+    if variant_obj.get("category") == "cancer":
+        variant_obj["matching_tiered"] = store.matching_tiered(
+            variant_obj, user_institutes(store, current_user)
```

Comment:
Answering here about you comment about this to be a "global" functionality. The matching tiered variants are collected only for cases in institutes the user is allowed to see (see the user_institutes used here). That should be OK right?

Reply:
Good; I missed that - I'm still not sure that we decided anything about sharing beyond general sentiments that it would be good to share, but then this lands on the safe side of that. 👍